Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -556,94 +548,3 @@ extension NewsTableViewController: UIViewControllerTransitioningDelegate {
         return DimPresentAnimationController()
     }
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f33780a953aef52ff2c255c7d8a8ee7464dd6567/Headlines/src/Controllers/NewsTableViewController.swift#L550"></span>File should contain 400 lines or less: currently contains 550<br />
<code>file_length</code> <code>NewsTableViewController.swift:550</code></td>
    </tr>
  </tbody>
</table>